OasisDEX/oasis-borrow
# 🧿 Ajna empty pools handling

# [Ajna empty pools handling](https://app.shortcut.com/oazo-apps/story/9825)

![image](https://github.com/OasisDEX/oasis-borrow/assets/16230404/3d0c1f73-c267-430c-a2a6-ce460cf09d54)
  
## Changes 👷‍♀️

- Added tooltips to not available max LTV and max multiple,
- hidden dynamic max LTV from content card and order information in Ajna Borrow and Multiply UI.
  
## How to test 🧪

- Run Product Hub PATCH,
- look for tooltips on empty pools on Product Hub,
- check if dynamic max LTV is hidden on open flows.
